pkg/logs/tail_test.go

```diff
@@ -0,0 +1,323 @@
+package logs
+
+import (
+	"io/ioutil"
+	"net/http"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+const responseFile = "testdata/response.json"
+
+func TestValidateTimes(t *testing.T) {
+	dur, _ := time.ParseDuration("10s")
+	for i, test := range []struct {
+		in  cfg
+		exp cfg
+	}{
+		{
+			in:  cfg{from: 1601480668, to: 1601480768, searchPadding: dur},
+			exp: cfg{from: 1601480658, to: 1601480778, searchPadding: dur},
+		},
+		{
+			in:  cfg{from: 1601480668, to: 1601480768},
+			exp: cfg{from: 1601480668, to: 1601480768},
+		},
+		{
+			in:  cfg{searchPadding: dur},
+			exp: cfg{searchPadding: dur},
+		},
+	} {
+		c := TailCommand{cfg: test.in}
+		c.adjustTimes()
+		if equal := reflect.DeepEqual(test.exp, c.cfg); !equal {
+			t.Errorf("#%d: validateTimes mismatch: got: %#+v  want: %#+v", i, c.cfg, test.exp)
```

Comment:
Presumed this was an old name?
```suggestion
			t.Errorf("#%d: adjustTimes mismatch: got: %#+v  want: %#+v", i, c.cfg, test.exp)
```

Reply:
Yes, and will also change the test name.